### ISSUE TITLE
smart picker seems to be broken

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#43478]
Seems to work but the smart picker seems to be broken 😢

https://github.com/nextcloud/server/assets/42591237/3ca2c572-2bbb-4fdd-9623-efa367ef2ff5

_Originally posted by @szaimen in https://github.com/nextcloud/server/issues/41879#issuecomment-1935663005_

**Comment by ShGKme** [comment]
I cannot reproduce on stable28 and master...

![image](https://github.com/nextcloud/server/assets/25978914/705e964e-ad36-43da-aa86-e5df741b2ae6)

**Comment by susnux** [comment]
Me neither. (master and stable28)

**Comment by ShGKme** [comment]
Tribute items have global styling. It means, if there are several apps with different `@nextcloud/vue` versions, it is random which styles will be applied to the all autocompletes on the page.
